types.go

```diff
@@ -599,12 +599,20 @@ type DashboardKey struct {
 
 // AlertSettings represents an alert configuration.
 type AlertSettings struct {
-	EscalationType      string              `json:"escalationType,omitempty"`
-	RunBasedEscalation  RunBasedEscalation  `json:"runBasedEscalation,omitempty"`
-	TimeBasedEscalation TimeBasedEscalation `json:"timeBasedEscalation,omitempty"`
-	Reminders           Reminders           `json:"reminders,omitempty"`
+	EscalationType              string                      `json:"escalationType,omitempty"`
+	RunBasedEscalation          RunBasedEscalation          `json:"runBasedEscalation,omitempty"`
+	TimeBasedEscalation         TimeBasedEscalation         `json:"timeBasedEscalation,omitempty"`
+	Reminders                   Reminders                   `json:"reminders,omitempty"`
+	ParallelRunFailureThreshold ParallelRunFailureThreshold `json:"parallelRunFailureThreshold,omitempty"`
```

Comment:
We need to update the BE to put [this](https://github.com/checkly/checkly-backend/pull/5138/commits/4958644eb1cc4882933e2b12ab7afb50f3faa059) back for public API as I removed it yesterday so that we can test it locally without exposing it

Reply:
@shiini2 ah good, i can open a PR tomorrow!